source/PowerAdjusterView.mc

```diff
@@ -255,12 +258,11 @@ class PowerDataField extends Ui.DataField {
      
     function onUpdate(dc) {
 	  View.onUpdate(dc);
-	  dc.setColor(Graphics.COLOR_BLACK, Graphics.COLOR_WHITE);
-	  dc.clear();
 	  var zone_label = "";
       var v = Ui.View.findDrawableById("value");
 	  var l = Ui.View.findDrawableById("label");
       if (my_zones.size() > 2) {
+     	  dc.setColor(Graphics.COLOR_WHITE, Graphics.COLOR_BLACK);
           l.setColor(Gfx.COLOR_WHITE);
```

Comment:
https://developer.garmin.com/downloads/connect-iq/monkey-c/doc/Toybox/WatchUi/DataField.html#getBackgroundColor-instance_method
This can be used to determine if the rest of the watch UI is white on black or black on white.